src/Makefile.am

```diff
@@ -214,7 +216,7 @@ libprotobuf_lite_la_SOURCES =                                  \
   google/protobuf/io/zero_copy_stream_impl_lite.cc
 
 libprotobuf_la_LIBADD = $(PTHREAD_LIBS) $(LIBATOMIC_LIBS)
-libprotobuf_la_LDFLAGS = -version-info 17:0:0 -export-dynamic -no-undefined
+libprotobuf_la_LDFLAGS = -version-info $(PROTOBUF_VERSION):0:0 -export-dynamic -no-undefined
```

Comment:
https://github.com/protocolbuffers/protobuf/commit/050fc9a43717193ffbccbd8ca5d19e1e5648c815#diff-480477e89f9b6ddafb30c4383dcdd705
For major release, we update the first (API change). For minor releases, we update the second (non-API change).

Reply:
BTW, the last update to 17 in 3.6.1 release seems suspicious.

Reply:
ok, I have updated the script to calculate this number from the given version. It should something like [MINOR_VERSION+OFFSET]:[MICRO_VERSION]:0, right?